conf/application.conf

```diff
@@ -209,7 +209,7 @@ optimizely {
 features {
   welsh-translation = true
   sca-wrapper-enabled = true
-  govuk-wallet-enabled = true
+  govuk-wallet-enabled = false
```

Comment:
Do we need this if we are no longer reading it?

Reply:
refactored